src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_vars.py

```diff
@@ -595,11 +596,15 @@ def method():
         del frame
 
 
-def change_attr_expression(frame, attr, expression, dbg, value=SENTINEL_VALUE):
+def change_attr_expression(frame, attr, expression, dbg, value=SENTINEL_VALUE, /, scope: Union[ScopeRequest, None]=None):
     """Changes some attribute in a given frame."""
     if frame is None:
         return
 
+    if scope is not None:
+        assert isinstance(scope, ScopeRequest)
```

Comment:
Why is the `assert` needed?

Reply:
I just copied this code from elsewhere, but it protects the `scope = scope.scope` to make sure the item is a scoperequest